### ISSUE TITLE
Select and input elements are different heights

### ISSUE THREAD

**Issue by jmsfwk** [reapit/elements#88]
**Describe the bug**
Input and select elements are different heights.

**To Reproduce**

```html
<div class="el-form-layout-has-margin el-form-layout">
  <div class="el-input-wrap">
    <div class="el-input-group">
      <input class="el-input" id="random-kduixd" type="text">
      <label for="random-kduixd" class="el-label">Input</label>    
    </div>
  </div>
  <div class="el-input-wrap">
    <div class="el-input-group">
      <select id="random-sbuh8b" type="number" class="el-select"></select>
      <label for="random-sbuh8b" class="el-label">Select</label>
    </div>
  </div>
</div>
```
https://codepen.io/jmsfwk/full/gOyzgvq

Steps to reproduce the behaviour eg:
1. View the HTML above
2. See error

**Expected behaviour**
Form elements to be the same height.

**Screenshots**
<img width="962" alt="Screenshot 2024-04-09 at 17 00 25" src="https://github.com/reapit/elements/assets/9892048/70cbbe4c-7338-4d06-aa0b-734713cbd250">

**Device (please complete the following information for web issues):**
 - Device: Mac
 - OS: OS X 10.15
 - Browser / Version: [Firefox 125.0b9](https://www.whatsmybrowser.org/b/RTXNH)

**Comment by willmcvay** [comment]
@jmsfwk Yep, that doesn't look right, will look into it in the morning.

**Comment by willmcvay** [comment]
@jmsfwk  This is in PR and will be released, probably tomorrow in the v4.3.0 release
